src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -0,0 +1,532 @@
+import React, { useEffect, useState } from 'react';
+
+import {
+  Alert,
+  Button,
+  Popover,
+  Text,
+  TextContent,
+  TextList,
+  TextListItem,
+  TextListVariants,
+  TextListItemVariants,
+  TextVariants,
+  Spinner,
+} from '@patternfly/react-core';
+import { ExclamationTriangleIcon, HelpIcon } from '@patternfly/react-icons';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+
+import ReleaseLifecycle from './../ImageOutput/ReleaseLifecycle';
+import ActivationKeyInformation from './../Registration/ActivationKeyInformation';
+import { AwsAccountId } from './../TargetEnvironment/Aws/AwsAccountId';
+
+import { RELEASES } from '../../../../constants';
+import { extractProvisioningList } from '../../../../store/helpers';
+import { useAppSelector } from '../../../../store/hooks';
+import { useGetOscapCustomizationsQuery } from '../../../../store/imageBuilderApi';
+import { useGetSourceListQuery } from '../../../../store/provisioningApi';
+import { useShowActivationKeyQuery } from '../../../../store/rhsmApi';
+import {
+  selectActivationKey,
+  selectArchitecture,
+  selectAwsAccountId,
+  selectAwsShareMethod,
+  selectBlueprintDescription,
+  selectBlueprintName,
+  selectCustomRepositories,
+  selectDistribution,
+  selectGcpAccountType,
+  selectGcpEmail,
+  selectGcpShareMethod,
+  selectProfile,
+  selectRegistrationType,
+} from '../../../../store/wizardSlice';
+import { useGetEnvironment } from '../../../../Utilities/useGetEnvironment';
+import { RepositoriesTable } from '../../../CreateImageWizard/formComponents/ReviewStepTables';
```

Comment:
This links to the original Wizard so if the user clicks on the number of Custom repositories on the Review step the whole Wizard crashes with `getState is not a function`

![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/d6a6c8e9-c03f-4463-9ad7-a846e179de9b)

```suggestion
import { RepositoriesTable } from './ReviewStepTables';
```

Reply:
woo stupit mistake